tutorials/vue/vue-select-onchange.md

```diff
@@ -0,0 +1,65 @@
+If you want to avoid having to use `v-model` with your `<select>` tags,
+you can use Vue's `v-on:change` directive, or just the shorthand `@change`. This functions
+similarly to the `v-model` directive. In every option tag, you must use the `value`
```

Comment:
I wouldn't say it's like `v-model` because it doesn't bind the value of the `select`. If you're using `v-model` and you change `this.selected` in JavaScript, that will also affect the `<select>` tag. I'd just get rid of the `v-model` sentence.